frontend/tests/features/collection/revision.test.ts

```diff
@@ -31,7 +31,7 @@ describe("Collection Revision @loggedIn", () => {
    * TODO(#5666): Enable this test once #5666 is resolved
    * https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/gh/chanzuckerberg/single-cell-data-portal/5666
    */
-  test.skip("starts a revision", async ({ page }) => {
+  test("starts a revision", async ({ page }) => {
```

Comment:
@tihuan re-added these two tests but it's failing 😞 

not obvious to me where the issue is since the smoke test initialization is now using the test user acct, and this e2e test uses (or appears to use) the same test user credentials...

Reply:
I know you specifically did not request for me to make this change as part of the issue but it really did _appear_ that it should be that simple (to my eye)

I can revert, and simply make the smoke test db initialization change, and then you can figure out how to reinstate the tests if you know what I'm missing

Reply:
Hey Dan! Thanks so much for enabling the test! That totally makes sense and serves as a way to ensure data seeding works 😆 

So far it seems like the smoke test account is not seeing any writable published collections it can pick from 🤔 

<img width="1057" alt="Screenshot 2023-10-10 at 9 46 21 AM" src="https://github.com/chanzuckerberg/single-cell-data-portal/assets/6309723/44e61ce7-6fcf-438a-a3b8-384fce0cbd2c">

<img width="1157" alt="Screenshot 2023-10-10 at 9 44 47 AM" src="https://github.com/chanzuckerberg/single-cell-data-portal/assets/6309723/9d66a84b-6011-47d7-8b8f-25233477ae5d">

And if it's helpful, maybe it'd help to manually log into the smoke test account in RDEV to check it yourself too!

Another thing is that e2e test result is stored under GHA summary, and you can download it under "logged in test result" like so:

<img width="1161" alt="Screenshot 2023-10-10 at 9 43 17 AM" src="https://github.com/chanzuckerberg/single-cell-data-portal/assets/6309723/5b892d71-29de-4dd6-bd5d-fb419b3d887e">

and unzipping the files will give you this directory:

<img width="751" alt="Screenshot 2023-10-10 at 9 43 35 AM" src="https://github.com/chanzuckerberg/single-cell-data-portal/assets/6309723/729b126c-5f4a-4318-9d93-5ee3e89df7e2">

The trace.zip is especially useful, to use it, please check out instructions here!

https://github.com/chanzuckerberg/single-cell-data-portal/tree/main/frontend/tests

<img width="1187" alt="Screenshot 2023-10-10 at 9 49 09 AM" src="https://github.com/chanzuckerberg/single-cell-data-portal/assets/6309723/a82e8773-4abd-421a-862a-f61b000e35ee">

Reply:
Thanks Timmy! I'll check this out

Reply:
Perfect! Thanks so much for your help with this, Dan 🙏 !

Reply:
Ah I see. The issue here is we need additional improvements to the seeding script _in addition to_ changing the existing collections to belong to the test user account rather than the func test account.